task/handler.go

```diff
@@ -43,8 +43,10 @@ func KoboldHandler(ctx context.Context, cache string, g model.TaskGroup, runner
 		g.DestBranch.String = g.RepoUri.Ref
 		g.DestBranch.Valid = true
 	}
-
-	msg = "chore(kobold): update image refs"
+	msg, err = GetCommitMessage(changes)
```

Comment:
I think we could make changes in a slice of structs with structured data so we don't have to use regex.

Reply:
I guess here the type needs to be changed to some exported struct type: https://github.com/bluebrown/kobold/blob/a92a2d2fce1c2836d5fc74bd391eb18e4bfc8595/krm/filter.go#L53